### PR TITLE
chore: Add issue template for general cluster scoped resource request

### PR DESCRIPTION
**Pull request by tumido**
Resolves: https://github.com/operate-first/support/issues/64

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/request_cluster_resources.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+title: ""
+labels: onboarding
+assignees: ""
+---
+
+### Requested actions
+
+#### Operator request
+
+If you want to request an operator to be deployed either with all namespaces visibility or with a single namespace visibility, please state here which operator you are interested in. Please point us to the any documentation to this operator directly.
```
Comment (anishasthana) [inline .github/ISSUE_TEMPLATE/request_cluster_resources.md:13]: `Please point us to the any documentation to this operator directly.` -> `Please point us to any documentation about this operator directly`

File: .github/ISSUE_TEMPLATE/request_cluster_resources.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+#### Resource request
+
+Please list all the resources you want to have deployed. Please list only the resources which can't be deployed by your project-admins. If possible, please include the resource manifest as a link to upstream definition or as an attachment file.
+
+Requesting there resources to be deployed:
```
Comment (anishasthana) [inline .github/ISSUE_TEMPLATE/request_cluster_resources.md:19]: /s/there/these

**Review by anishasthana — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/request_cluster_resources.md (hunk 1 above)
Comment (tumido) [reply .github/ISSUE_TEMPLATE/request_cluster_resources.md:13]: applied

File: .github/ISSUE_TEMPLATE/request_cluster_resources.md (hunk 2 above)
Comment (tumido) [reply .github/ISSUE_TEMPLATE/request_cluster_resources.md:19]: applied :+1:

File: .github/ISSUE_TEMPLATE/request_cluster_resources.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+title: ""
+labels: onboarding
+assignees: ""
+---
+
+### Requested actions
+
+#### Operator request
+
+If you want to request an operator to be deployed either with all namespaces visibility or with a single namespace visibility, please state here which operator you are interested in. Please point us to any documentation about this operator directly
```
Comment (HumairAK) [inline .github/ISSUE_TEMPLATE/request_cluster_resources.md:13]: I think we can be a bit more forward here, suggestion: 

>If you want to request an operator, please provide the following: 
>- Name of the Operator
>- Operator version
>- Should the Operator be visible to a _Single Namespace_ or _All Namespaces_ 
>- Any documentations about this operator (if applicable)

Comment (tumido) [reply .github/ISSUE_TEMPLATE/request_cluster_resources.md:13]: Applied, thanks :+1:

**Review by HumairAK — APPROVED** [review]

**Review by anishasthana — APPROVED** [review]

**Pull request merged**